site/en/docs/extensions/reference/storage/index.md

```diff
@@ -34,70 +74,77 @@ API. For example:
-});
+  </web-tab>
+  <web-tab title="storage.sync">
 
-chrome.storage.local.get(['key'], function(result) {
-  console.log('Value currently is ' + result.key);
-});
-```
+  ```js
+  chrome.storage.sync.set({ key: value }).then(() => {
+    console.log("Value is set to " + value);
+  });
 
-When using `storage.sync`, the stored data will automatically be synced to any Chrome browser that
-the user is logged into, provided the user has sync enabled.
+  chrome.storage.sync.get(["key"]).then((result) => {
+    console.log("Value currently is " + result.key);
+  });
+  ```
 
-When Chrome is offline, Chrome stores the data locally. The next time the browser is online, Chrome
-syncs the data. Even if a user disables syncing, `storage.sync` will still work. In this case, it
-will behave identically to `storage.local`.
+  </web-tab>
+  <web-tab title="storage.session">
 
-{% Aside 'warning' %}
+  ```js
+  chrome.storage.session.set({ key: value }).then(() => {
+    console.log("Value is set to " + value);
+  });
 
-Confidential user information should not be stored! The storage area isn't encrypted.
+  chrome.storage.session.get(["key"]).then((result) => {
+    console.log("Value currently is " + result.key);
+  });
+  ```
+
+  </web-tab>
+</web-tabs>
 
-{% endAside %}
 
-The `storage.managed` storage is read-only.
+To learn more about the `managed` storage area, see [Manifest for storage areas][manifest-storage].
 
-## Storage and throttling limits
 
-`chrome.storage` is not a big truck. It's a series of tubes. And if you don't understand, those
+## Storage and throttling limits {: #storage-and-throttling-limits}
+
+The Storage API is not a big truck. It's a series of tubes. And if you don't understand, those
 tubes can be filled, and if they are filled when you put your message in, it gets in line, and it's
 going to be delayed by anyone that puts into that tube enormous amounts of material.
 
-For details on the current limits of the storage API, and what happens when those limits are
-exceeded, please see the quota information for [sync][prop-sync] and [local][prop-local].
+For details on storage area limitations and what happens when they are exceeded, see the quota information for [`sync`][prop-sync], [`local`][prop-local], and [`session`][prop-session].
 
-## Examples
+## Use cases {: #examples}
 
-The following sections demonstrate how to use `chrome.storage` to address some common use cases.
+The following sections demonstrate some common use cases for the Storage API.
 
 ### Synchronous response to storage updates
 
-If you're interested in tracking changes made to a data object, you can add a listener to its
-`onChanged` event. Whenever anything changes in storage, that event fires. Here's sample code to
-listen for saved changes:
+To track changes made to storage, you can add a listener to its `onChanged` event. When anything changes in storage, that event will fire. The sample code listens for these changes:
```

Comment:
```suggestion
To track changes made to storage, you can add a listener to its `onChanged` event. When anything changes in storage, that event fires. The sample code listens for these changes:
```

Reply:
This is non-obvious, but it will probably blow your mind. Most instances of 'will' in computer documentation can be removed. I worked for 14 years at a place that specified this in its style guide. No one ever noted its absence or filed a bug that they didn't understand.